support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/ticket/registry/TicketHolder.java

```diff
@@ -0,0 +1,53 @@
+package org.apereo.cas.ticket.registry;
+
+import com.datastax.driver.mapping.annotations.PartitionKey;
+import com.datastax.driver.mapping.annotations.Table;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+/**
+ * This is {@link TicketHolder}
+ *
+ * @since 6.1.0
+ */
+@Table(name = "ticket", writeConsistency = "LOCAL_QUORUM", readConsistency = "ONE")
+public class TicketHolder {
```

Comment:
Are `writeConsistency = "LOCAL_QUORUM"` and `readConsistency = "ONE"` things that should be controlled via CAS settings?